### DISCUSSION TITLE
Is possible to use a  `link-external-icon`  without affect `<pre><code>` tags?

### DISCUSSION THREAD

**Discussion by jbkunst** [quarto-dev/quarto-cli#3755] [Q&A]
Hi all,

Is not clear if is possible use  `link-external-icon` feature without affect code when using `code-link: true`. Clearly this Is not a issue/bug, but it would a nice feature to separate this options somehow imho.

![image](https://user-images.githubusercontent.com/56481/209453018-0a8f4df7-bc72-43c8-a613-fc45ce24879d.png)

```
---
title: "Links external"
link-external-icon: true
link-external-newwindow: true
code-link: true
---

This is a [link to quarto website](https://quarto.org/)

 ```{r}
 #| message: false
 #| warning: false
 # a code with links
 library(tidyverse)
 
 tibble(x = c(1, 2))
```{end r}

```

**Comment by dragonstyle** [comment]
Thanks for reporting this- I agree that this seems pretty undesirable and we should be screening these out (we already screen other links out elsewhere). Would you mind transferring this to an issue and I'll try to get it fixed for the 1.3 release?
